applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -488,6 +512,31 @@ static void fix_rep_wk(struct k_work *work)
 		}
 	}
 
+	err = nrf_modem_gnss_read((void *)&nmea, sizeof(nmea), NRF_MODEM_GNSS_DATA_NMEA);
+	if (err) {
+		LOG_WRN("Failed to read GNSS NMEA data, error %d", err);
+	} else {
+		/* Report to nRF Cloud by best-effort */
+		if (nrf_cloud_ready && location_signify) {
+			err = date_time_now(&ts_ms);
```

Comment:
`date_time_now()` seems like a bit overhead to me to calculate a timeout, `k_uptime_get(0` should be enough (time in ms since system boot, much simpler function).

Reply:
this ts_ms is used in later call of nrf_cloud_rest_send_location(), which requires UTC Unic epoch time in ms

Reply:
ok, nevermind then